lib/3scale_toolbox/commands/copy_command/copy_service.rb

```diff
@@ -1,11 +1,26 @@
 require 'cri'
+require '3scale/api'
 require '3scale_toolbox/base_command'
 
 module ThreeScaleToolbox
   module Commands
     module CopyCommand
       class CopyServiceSubcommand < Cri::CommandRunner
         include ThreeScaleToolbox::Command
+        include ThreeScaleToolbox::Remotes
+
+        attr_reader :source, :destination, :system_name, :service_id,
+                    :source_remote, :copy_remote,
+                    :source_service, :copy_service,
```

Comment:
This looks a lot like `source` and `copy` are the same kind of object.

If they are, it would be nicer to have just `source` and `destination` and methods `metrics` and others implemented there.

You know, it is a *lot* of instance variables. I'd try to not have so many and rather group them into more meaningful objects.